buddyboss/buddyboss-platform
# Fix: Last name - Privacy settings not working for forum the Display Name

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2526  .

### How to test the changes in this Pull Request:

1. Go to Setting->profile->Display Name Format->select "First name and Last name"
2. Now Go to front side User Edit Profile -> Change Lastname privacy "only me"
3. Login with other user and check the forum and see the previous changed user reply or discussion display name


### Proof Screenshots or Video

https://www.loom.com/share/e188591291be405f8d1374e2a7dc8e2b

### Other information:
